### ISSUE TITLE
Compound master: keyboard navigation

### ISSUE THREAD

**Issue by jhou-pro** [fieldenms/tg#1043]
### Description
Compound masters should have proper mechanism of keyboard navigation by means of using `Tab` and `Shift + Tab` keys and ability to navigate to menu using `F4`. The order of navigatable elements should be the following: title bar buttons, menu (with `arrow`-driven navigation), main content. Invisible (unactionable) elements should not be navigatable -- this means that only buttons, editors, menu items should gain focus.

### Expected outcome
Predictable and usable keyboard navigation in compound masters.
